test/test_metadata.py

```diff
@@ -131,6 +131,21 @@ def test_metadata_undelete(self):
         self.assertIn("single1", self.metadata)
         self.assertNotIn("single1", self.metadata.deleted_tags)
 
+    def test_normalize_tag(self):
+        self.assertEqual('sometag', Metadata.normalize_tag('sometag'))
+        self.assertEqual('sometag', Metadata.normalize_tag('sometag:'))
```

Comment:
Since `Metadata.normalize_tag()` is using `str.rstrip()`, it will strip all occurences of `:` from the end.

```python
>>> 'comment:::'.rstrip(':')
'comment'
```

This case isn't tested here. Not sure whether it should or if removing only one occurence is the way to go.

Reply:
I think it should remove all of them, especiall since we introduced the `comment:{language}:{description}` syntax. I'll add a test